plugins/commands/evaluator/src/main/kotlin/EvaluatorCommand.kt

```diff
@@ -148,19 +148,19 @@ class EvaluatorCommand : OrtCommand(
 
     private val packageCurationsFile by option(
         "--package-curations-file",
-        help = "A file containing package curations. This replaces all package curations contained in the given ORT " +
-            "result file with the ones present in the given file and, if enabled, those from the package " +
-            "configuration."
+        help = "A file containing package curations to replace the package curations in the given ORT file. The " +
```

Comment:
Hmm, the docs *still* seem to refer to package curations, so I'm unsure about the intention of the change.

Reply:
Ah, I guess you meant "package configuration" instead of "package curations" in the commit message.

All in all, I find the existing wording to be more clear, and I'd propose to just change "package configuration" to "repository configuration" in the last sentence.